src/libraries/System.Runtime.CompilerServices.Unsafe/src/include/netstandard2.0/coreassembly.h

```diff
@@ -1,20 +0,0 @@
-#define CORE_ASSEMBLY "System.Runtime"
-
-// Metadata version: v4.0.30319
-.assembly extern CORE_ASSEMBLY
-{
-  .publickeytoken = (B0 3F 5F 7F 11 D5 0A 3A )                         // .?_....:
-  .ver 4:0:0:0
-}
-
-/*  ILasm currently complains when seeing a [netstandard]System.Object
```

Comment:
Since you're deleting this comment, can you fix it by setting CoreAssembly to netstandard on `netstandard2.0` TargetFramework?